x-pack/test/functional/apps/lens/group3/metric.ts

```diff
@@ -0,0 +1,228 @@
+        dimension: 'lnsMetric_breakdownByDimensionPanel > lns-empty-dimension',
+        operation: 'terms',
+        field: 'ip',
+      });
+
+      await PageObjects.lens.waitForVisualization('mtrVis');
+
+      expect(await getMetricData()).to.eql([
+        {
+          title: '97.220.3.248',
+          subtitle: 'Average of bytes',
+          extraText: '19.76K',
+          value: '19.76K',
+          color: 'rgba(245, 247, 250, 1)',
+          showingBar: false,
+        },
+        {
+          title: '169.228.188.120',
+          subtitle: 'Average of bytes',
+          extraText: '18.99K',
+          value: '18.99K',
+          color: 'rgba(245, 247, 250, 1)',
+          showingBar: false,
+        },
+        {
+          title: '78.83.247.30',
+          subtitle: 'Average of bytes',
+          extraText: '17.25K',
+          value: '17.25K',
+          color: 'rgba(245, 247, 250, 1)',
+          showingBar: false,
+        },
+        {
+          title: '226.82.228.233',
+          subtitle: 'Average of bytes',
+          extraText: '15.69K',
+          value: '15.69K',
+          color: 'rgba(245, 247, 250, 1)',
+          showingBar: false,
+        },
+        {
+          title: '93.28.27.24',
+          subtitle: 'Average of bytes',
+          extraText: '15.61K',
+          value: '15.61K',
+          color: 'rgba(245, 247, 250, 1)',
+          showingBar: false,
+        },
+        {
+          title: 'Other',
+          subtitle: 'Average of bytes',
+          extraText: '5.72K',
+          value: '5.72K',
+          color: 'rgba(245, 247, 250, 1)',
+          showingBar: false,
+        },
+      ]);
+
+      await PageObjects.lens.openDimensionEditor(
+        'lnsMetric_maxDimensionPanel > lns-empty-dimension'
+      );
+
+      await PageObjects.lens.waitForVisualization('mtrVis');
+
+      expect((await getMetricData())[0].showingBar).to.be(true);
+
+      await PageObjects.lens.closeDimensionEditor();
+      await PageObjects.lens.removeDimension('lnsMetric_maxDimensionPanel');
+    });
+
+    it('should filter by click', async () => {
+      expect((await filterBar.getFiltersLabel()).length).to.be(0);
+      const title = '93.28.27.24';
+      await clickMetric(title);
+
+      await PageObjects.common.sleep(1000);
+
+      retry.try(async () => {
+        const labels = await filterBar.getFiltersLabel();
+        expect(labels.length).to.be(1);
+        expect(labels[0]).to.be(`ip: ${title}`);
+      });
+
+      await filterBar.removeAllFilters();
+      await PageObjects.lens.waitForVisualization('mtrVis');
+    });
+
+    it('applies static color', async () => {
+      await PageObjects.lens.openDimensionEditor(
+        'lnsMetric_primaryMetricDimensionPanel > lns-dimensionTrigger'
+      );
+
+      const colorPicker = await testSubjects.find('euiColorPickerAnchor');
+      await colorPicker.type('#000000');
+
+      await PageObjects.common.sleep(1000); // allow new tiles to render
```

Comment:
These sleeps should be replaced by `PageObjects.lens.waitForVisualization` - this is the general helper user in all the functional tests to wait for a re-render. No need to have a different strategy for these tests